### PR TITLE
Create event for alterconf

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/S2015/S15-event-alterconf.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Date: 2015-07-11 9:00
+Category: Events
+Tags: social
+Slug: alterconf-toronto-2015
+Author: Fatema Boxwala
+Summary: Hitch a ride with WiCS to AlterConf in Toronto!
+
+WiCS has chartered a bus to transport Univeristy of Waterloo students to
+AlterConf on Saturday July 11 2015! Space is limited so be sure to
+[register](https://goo.gl/86ILSn) and RSVP to the FaceBook event below.
```
Comment (ameesha) [inline content/S2015/S15-event-alterconf.md]: There's no link to the Facebook event. Both links are to the registration form (and one to the alterconf site).

File: content/S2015/S15-event-alterconf.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+iTitle: WiCS is going to AlterConf Toronto!
```
Comment (ameesha) [inline content/S2015/S15-event-alterconf.md]: That "i" before Title looks like a typo.

File: content/S2015/S15-event-alterconf.md (hunk 1 above)
Comment (annalorimer) [inline content/S2015/S15-event-alterconf.md]: This should say the link to the Google form. The FB group is private and can't be joined by anyone not in the WiCS FB group.

**Comment by ameesha** [comment]
Looks good, merging.

**Pull request merged**